### DISCUSSION TITLE
FLUX banding/striping in image

### DISCUSSION THREAD

**Discussion by BrechtCorbeel** [Comfy-Org/ComfyUI#4237] [General]
I asked on the FLUX model Github too, but since it's likely a node setting or model change I am asking here too:

Anyone know how to get rid of this sort of banding/striping happening in the image?

![image](https://github.com/user-attachments/assets/c9b4d2b3-c737-4d2c-a825-bc12a801d089)

**Comment by ltdrdata** [comment]
What is your workflow?

**Reply by BrechtCorbeel** [reply]
> What is your workflow?

I currently changed to the DEV model from hugging face, these results is what I got from using the ones from the comfyui browser.

Once I switched it cleared up

**Reply by ltdrdata** [reply]
Do you meaning it is the model problem?

**Reply by BrechtCorbeel** [reply]
> Do you meaning it is the model problem?

It was a model problem for me or maybe a missmatch somewhere with the schnell model, but when I switched to the [Dev] model from huggingface everything worked fine.

**Comment by Vagrin** [comment]
I've gotten banding with every Flux model when large 2K images are generated. I think it's a problem with Flux not any sort of upscaler/tiler because you'll get it if you just ask Flux to generate an image that large. No one has come across a fix?

**Comment by FunnyFinger** [comment]
I have the same problem, but vertical, maybe I'm using wide aspect ratio instade of portrait. Also this happens only when upscaling, but not always. It's seems that if I use a combined lora wieght greater than 1, I most likely will get banding. Also I noticed that I'll get some noise in blurred background areas when upscaling. I'm using ultimate upscaler with the same Flux 1 dev model, changing the sampler or scheduler wont affect the result. At this point, only Loras are affecting my results.

**Comment by Sana-A-E** [comment]
Not a Comfy issue but Flux issue:
https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/1712

**Reply by Vagrin** [reply]
Not my experience. I got the same banding in ComfyUI and my friend who only uses ComfyUI says it's there as well. [shrug] We'll see how it all works itself out.

**Comment by jozefcoder** [comment]
> I asked on the FLUX model Github too, but since it's likely a node setting or model change I am asking here too:
> 
> Anyone know how to get rid of this sort of banding/striping happening in the image?
> 
> ![image](https://private-user-images.githubusercontent.com/68238477/355318312-c9b4d2b3-c737-4d2c-a825-bc12a801d089.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ4MjM2MDUsIm5iZiI6MTczNDgyMzMwNSwicGF0aCI6Ii82ODIzODQ3Ny8zNTUzMTgzMTItYzliNGQyYjMtYzczNy00ZDJjLWE4MjUtYmMxMmE4MDFkMDg5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjIxVDIzMjE0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc2ZGJmOTJiYzllODQ0Nzc4YzBmMDY0MjBkMDk2NGVhNDEzMzI5NDczN2QwYmU4MGI3MDdjM2Y3ODdkYjdhNDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.QtJRATE8_fcYcSAENNFS3mKNhjh05a8AtBrpeFX7r4g)

I know how to solve this problem for up to 4-5 megapixels.
![FantasyGirl](https://github.com/user-attachments/assets/0a3bc6a9-0fe5-4a4b-9e18-7c54021d5501)

**Comment by jozefcoder** [comment]
The solution is in pdf here, not free https://ko-fi.com/s/2f2d90c749
and copy here https://www.patreon.com/jpaedu/shop/mastering-ai-image-generation-with-flux-810873

But there is much more addressed, please see the description.

**Reply by Ainaemaet** [reply]
Get out of here with that nonsense - what a way to make a bad name for yourself.

**Comment by jozefcoder** [comment]
See the solution in https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/1712

**Comment by vararth** [comment]
@jozefcoder gatekeeping solutions to open source developments is a fun idea - but restricts freedom as well. If you have a solution - please post it as well rather than just self promote..

For others - no real solution exists, you should use the correct resolution for your images - many LoRA's increase the stripes bug. Additionally, once you have a clean image - you can upscale with the Upscaler ControlNet and the UltimateSDUpscale method - but remember to use lower LoRA weights, low denoise and low CFG for this to avoid stripes. It is not a foolproof solution but does work most of the time to avoid the stripes / grid / lines .

**Comment by trollver9000** [comment]
You are wrong, solution exists, DyPE fixes it its available for comfy, tested it just right now and banding disappears. https://github.com/wildminder/ComfyUI-DyPE/issues/3

**Reply by vararth** [reply]
> You are wrong, solution exists, DyPE fixes it its available for comfy, tested it just right now and banding disappears. [wildminder/ComfyUI-DyPE#3](https://github.com/wildminder/ComfyUI-DyPE/issues/3)

Considering the project is 3 weeks old and I made the comment in April - sure, I can accept solutions that are posted since then. Also noticed stripes disappear anyway when using nunchaku flux - at least bands and stripes are far fewer through the nunchaku inferencing compared to the full model and other fp16, fp8 quantized models.
